src/ceph-disk/ceph_disk/main.py

```diff
@@ -2362,17 +2362,18 @@ def create_key(self):
         cluster = self.args.cluster
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
-        command_check_call(
+        command_with_stdin(
```

Comment:
@smithfarm @tchaikov  Does every piece of code in every backport have to reference some commit in a later branch?